External/Poly2Tri/Triangulation/TriangulationContext.cs

```diff
@@ -36,7 +36,7 @@ abstract class TriangulationContext {
 		public TriangulationDebugContext DebugContext { get; protected set; }
 
 		public readonly List<DelaunayTriangle> Triangles = new List<DelaunayTriangle>();
-		public readonly List<TriangulationPoint> Points = new List<TriangulationPoint>(200);
+		public readonly List<TriangulationPoint> Points = new List<TriangulationPoint>();
```

Comment:
What's the reason for removing the preallocation of the list to a bigger size?

Reply:
In the majority of cases triangulation is used for single faces, which are usually around 6 points.